apps/test/unit/gamelab/GameLabSpriteTest.js

```diff
@@ -205,4 +205,170 @@ describe('GameLabSprite', function () {
       });
     });
   });
+
+  describe('collision types using AABBOps', function () {
+    var sprite, spriteTarget;
+
+    beforeEach(function () {
+      // sprite in to the left, moving right
+      // spriteTarget in to the right, stationary
+      sprite = createSprite(281, 100, 20, 20);
+      spriteTarget = createSprite(300, 100, 20, 20);
+      sprite.velocity.x = 3;
+      spriteTarget.velocity.x = 0;
+    });
+
+    it('stops movement of colliding sprite when sprites bounce', function () {
+      // sprite stops moving, spriteTarget moves right
+      expect(sprite.velocity.x).to.equal(3);
+      expect(spriteTarget.velocity.x).to.equal(0);
```

Comment:
This initial pair of assertions (the starting velocities) are the same for every test.  You can extract them into the `beforeEach` function.